vispy/ext/osmesa.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# vispy: testskip
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+
+"""A ctypes-based API to OSMesa
+"""
+from __future__ import print_function
+import os
+import ctypes
+from ctypes import c_int as _c_int, c_uint as _c_uint, c_void_p
+from vispy.gloo import gl
```

Comment:
things in `ext` should be independent of `vispy`, so you should just sub in the constants here like:
```
GL_RGBA = 6408
GL_UNSIGNED_BYTE = 5121
```